visualization/FairnessDashboard/src/Localization/zh-TW.json

```diff
@@ -0,0 +1,154 @@
+{
+  "loremIpsum": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+  "defaultClassNames": "類別 {0}",
+  "defaultFeatureNames": "敏感性特徵 {0}",
+  "defaultSingleFeatureName": "敏感性特徵",
+  "defaultCustomMetricName": "自訂指標 {0}",
+  "accuracyTab": "成效公平性",
+  "opportunityTab": "商機公平性",
+  "modelComparisonTab": "模型比較",
+  "tableTab": "詳細資料檢視",
+  "dataSpecifications": "資料統計資料",
+  "attributes": "屬性",
+  "singleAttributeCount": "1 個敏感性特徵",
+  "attributesCount": "{0} 個敏感性特徵",
+  "instanceCount": "{0} 個執行個體",
+  "close": "關閉",
+  "calculating": "正在計算...",
+  "accuracyMetric": "成效指標",
+  "errorOnInputs": "輸入發生錯誤。敏感性特徵目前必須是類別目錄值。請將值對應至組界類別，然後重試。",
+  "Accuracy": {
+    "header": "要如何測量成效?",
+    "modelMakes": "模型會建立",
+    "modelsMake": "模型會建立",
+    "body": "您的資料包含 {0} 個標籤和 {2} 個 {1} 預測。根據這些資訊，建議您使用下列指標。請從清單中選取一個指標。",
+    "binaryClassifier": "二元分類器",
+    "probabalisticRegressor": "概率迴歸輸入變數",
+    "regressor": "迴歸輸入變數",
+    "binary": "二元",
+    "continuous": "連續"
+  },
+  "Parity": {
+    "header": "依據差距進行測量的公平性",
+    "body": "差距指標會在所選功能間量化模型行為的變化。有兩種差距指標: 即將推出更多...."
+  },
+  "Header": {
+    "title": "Fairlearn",
+    "documentation": "文件"
+  },
+  "Footer": {
+    "back": "上一步",
+    "next": "下一步"
+  },
+  "Intro": {
+    "welcome": "歡迎使用",
+    "fairlearnDashboard": "Fairlearn 儀表板",
+    "introBody": "Fairlearn 儀表板可讓您評定模型成效與公平性之間的權衡",
+    "explanatoryStep": "若要設定評定，您必須指定敏感性特徵和成效指標。",
+    "getStarted": "立即開始",
+    "features": "敏感性特徵",
+    "featuresInfo": "敏感性特徵會用於將資料分割成群組。這些群組的模型公平性會以差距指標測量。差距指標會量化模型行為在各群組之間的差異。",
+    "accuracy": "成效指標",
+    "accuracyInfo": "成效指標可用於評估模型的整體品質，以及每個群組中的模型品質。各群組成效指標極端值之間的差異，會在成效中回報為差距。"
+  },
+  "ModelComparison": {
+    "title": "模型比較",
+    "howToRead": "如何閱讀此圖表",
+    "lower": "低於",
+    "higher": "高於",
+    "howToReadText": "此圖表會將每個 {0} 模型表示為可選取的點。X 軸代表 {1}，越{2}越好。Y 軸代表差距，越低越好。",
+    "insights": "見解",
+    "insightsText1": "此圖表顯示 {0} 與 {1} 模型的差距。",
+    "insightsText2": "{0} 的範圍為 {1} 到 {2}。差距的範圍為 {3} 到 {4}。",
+    "insightsText3": "最準確的模型達到 {1} 的 {0} 和 {2} 的差距。",
+    "insightsText4": "最低差距模型達到 {1} 的 {0} 和 {2} 的差距。",
+    "disparityInOutcomes": "預測中的差距",
+    "disparityInAccuracy": "{0} 中的差距",
+    "howToMeasureDisparity": "應如何測量差距?"
+  },
+  "Report": {
+    "modelName": "模型 {0}",
+    "title": "成效差距",
+    "globalAccuracyText": "為整體 {0}",
+    "accuracyDisparityText": "為 {0} 中的差距",
+    "editConfiguration": "編輯組態",
+    "backToComparisons": "多重模型檢視",
+    "outcomesTitle": "預測中的差距",
+    "minTag": "最小值",
+    "maxTag": "最大值",
+    "groupLabel": "子群組",
+    "underestimationError": "低於預測",
+    "underpredictionExplanation": "(已預測 = 0，True = 1)",
```

Comment:
False gets a translation, and True doesn't?

Reply:
I thought that was particularly odd...